meteor-app/imports/api/parts/parts.js

```diff
@@ -0,0 +1,44 @@
+import { Mongo } from 'meteor/mongo'
+import SimpleSchema from  'simpl-schema'
+
+import { createdAt, updatedAt } from '/imports/api/schema'
+
+const Parts = new Mongo.Collection('parts')
+
+export const PartsSchema = new SimpleSchema({
+  imageUrl: {
+    type: String,
+    defaultValue: '/public/images/logo-large.jpg',
+  },
+  retailPrice: {
+    type: Number,
+    label: 'Retail Price',
+    optional: false,
+  },
+  wholesalePrice: {
+    type: Number,
+    label: 'Wholesale price',
+    optional: false,
+  },
+  number: {
```

Comment:
I am generally a fan of short names (saves typing), but in this case it feels like it is out of context, and is too close to `Number`. Let's use `partNo` instead